.circleci/config.yml

```diff
@@ -521,8 +521,6 @@ jobs:
 
   ios_test_suite:
     executor: mac
-    macos: # Detox simulators require 10.1.0
-      xcode: '10.1.0'
```

Comment:
Will it be ok to leave the value undefined? Wouldn't we want to specify some specific version?

See the list at https://circleci.com/docs/2.0/testing-ios/#supported-xcode-versions.

Reply:
Here you can just override the default Xcode version for the executor, which is defined in the executor configuration: https://github.com/expo/expo/blob/cb86e366018bea825b0349aad19d77a683c412ef/.circleci/config.yml#L312-L314

So yes, that's ok 😉

Reply:
Ahh, ok, thanks, I thought it's a change in the line you've linked to. 😃